### ISSUE TITLE
Visual cue for grabbing and moving book when using keyboard (alt input )

### ISSUE THREAD

**Issue by terracoda** [phetsims/friction#142]
In #141 we discussed possible visual cues for the grab-move interaction.
We decided to 
- remove visual references to WASD keys as they are only needed when using a keyboard. The WASD cues are provided in PDOM and in Keyboard Shortcuts dialog.
- visually highlight the three important directions rather than all four.

@zepumph, please implement the visual cues below. Visual cues disappear after successful interaction, and are returned after the Reset All button is pressed, or the page is re-loaded.

**1. On focus of the grab button**, use the same visual cue from Balloons and Static Electricity:
<img width="829" alt="01-chem-book-focus-hint-no-arrows" src="https://user-images.githubusercontent.com/1902116/47899195-0f114800-de79-11e8-9fd3-92b40a164279.png">

<img width="829" alt="01-zoomed-in-focus-hint-no-arrows" src="https://user-images.githubusercontent.com/1902116/47899210-1b95a080-de79-11e8-8fba-0b388ecafaf3.png">

2. **Once grabbed,** show 3 pink arrows with the dashed grabbed focus outline:
![02-3-arrows-book](https://user-images.githubusercontent.com/1902116/47899256-47188b00-de79-11e8-9b36-25b136861cdf.png)

![02-3-arrows-zoomed-in-book](https://user-images.githubusercontent.com/1902116/47899320-77602980-de79-11e8-9024-95055aef773c.png)

**Comment by emily-phet** [comment]
@terracoda @zepumph These mockups look great! One request, for the grabbed state in the zoomed in view (the last image above), let's make the arrows larger (50-100% larger) and move the a little down so that the top of the downward facing arrow is at the first (top) row of blue particles.

**Comment by terracoda** [comment]
@emily-phet, I agree on larger arrows. I was going to ask if the arrows should be the same size as the the white arrows are in the zoomed-in view.

**Comment by zepumph** [comment]
The above commits all were working on the cue button. In https://github.com/phetsims/scenery-phet/issues/419 we moved the BASE cue node to common code, and I worked it into the general Grab button pretty easily. 

Now, with the arrows, we will mostly be directly in the friction repo, making these custom.

**Comment by zepumph** [comment]
The above commit added arrow cues to the macro book, but it also helped to demonstrate the problems being worked on in https://github.com/phetsims/friction/issues/144 and https://github.com/phetsims/scenery-phet/issues/421. It messed with the grab button focusHighlight. I am going to meet with @jonathanolson tomorrow to try to work out a nice general solution to apply to GrabButtonNode. Until then master looks very weird and broken (though it has cue arrows), but it still should logically work.

**Comment by zepumph** [comment]
Please ignore the commits above to the previous comment, they are mainly related to https://github.com/phetsims/scenery-phet/issues/421, though they do pertain to this issue some.

**Comment by zepumph** [comment]
This has been implemented. @terracoda please review here:

https://phet-dev.colorado.edu/html/friction/1.5.0-dev.22/phet/friction_all_phet.html

Note that the zoomed-in book arrows are a little different. This was the easiest to accomplish right now, because right now the atom circles actually end up on top of the arrows if I move the bottom one down any more. That can likely be fixed, but this was easy to do, so I thought I would let you look at it.

**Comment by emily-phet** [comment]
Looks good to me! 
I wish it wasn't so busy looking when the lower chemistry book (not zoomed in view) has focus to start, but I can't think of a way to improve it. With the physics book text and chemistry book text, the little zoom-in black box, focus highlight, and the 'space to grab' prompt, there's quite a lot happening in a small space - but I think this is minor and probably just fine.

**Comment by terracoda** [comment]
@zepumph, thanks. I was pretty sure you had implemented, but I was waiting for the review label to comment. @emily-phet beat me to it.

I agree it looks good, but I have one question for the design meeting..."What are we doing with the white arrow keys?"

Can we make them the same size as the pink ones (but right/left only) and then on grab add the third arrow and change the arrows to pink?

**Comment by terracoda** [comment]
As mentioned at the design meeting, if the white arrows the same size as the pink ones, and thus small enough to be covered by the visual grab hint, the transition between no focus, to focus, and then to grab might be quite smooth.

**Comment by zepumph** [comment]
I added back in the white arrows. They hide nicely behind the the grab button cue node. While I was doing this, I realized that all the arrows were being drawn pretty strangely, so I refactored them. They ended up a bit different than before, please review. If we want I can make them what they were, this was just easier in the moment.

**Comment by zepumph** [comment]
@terracoda anything else here?
